components/molecules/FeedbackWidget.js

```diff
@@ -188,32 +190,34 @@ export const FeedbackWidget = ({
                 ""
               )}
               <div className="layout-container text-white pb-4">
-                <button
+                <ActionButton
                   id="feedbackClose"
+                  dataCy="closeButton"
+                  dataTestId="closeButton"
+                  ariaLabel="Close the expanded feedback section"
+                  custom="flex float-right pt-2 font-body text-white flex mt-2.5 lg:mt-0 outline-none focus:outline-white-solid items-center"
+                  imageSource="/close-x.svg"
+                  imageAlt="Close button"
+                  imageSpanClass="text-xs leading-4 lg:text-sm underline ml-2 lg:leading-10"
+                  imageSpanText={t("close")}
```

Comment:
`aria-label` should only be used on elements that don't have text, or don't have a label associated with an image. In this case, there is a span inside the button with text, which accurately describes what the button does, so `aria-label` shouldn't be necessary here.